PlayerConnection.cs

```diff
@@ -131,7 +139,7 @@ public override string ToString()
                     res.m_AllowDebugging = 0 != int.Parse(playerSettings["debug"]);
                     if (playerSettings.ContainsKey("debuggerport"))
                         res.m_DebuggerPort = uint.Parse(playerSettings["debuggerport"]);
-
+                    res.m_ProjectName = playerSettings["projectname"];
```

Comment:
To be backwards compatible wrap this in a `if (playerSettings.ContainsKey("projectname"))`